docs/assets/main.scss

```diff
@@ -50,8 +50,10 @@ a {
   }
 }
 
-*:focus {
+*:focus-visible {
   outline-color: #757575;
+  outline:5px solid #757575; 
```

Comment:
Would you remove line 54? Because we already have `outline: 5px solid #757575;` above and `outline`  is a shorthand for `outline-width` `outline-style` `outline-color`, having `outline-color` defined separately is duplicate.

Reply:
ok